templates/channels/channel_claim.haml

```diff
@@ -252,6 +252,22 @@
                 -blocktrans
                   You can connect a messaging service from your Twilio account to benefit from <a href="https://www.twilio.com/copilot">Twilio Copilot features</a></br>
 
+    -if not user_org.is_connected_to_twilio_compliant_api
+      .claim-row.twilio-compliant-api-channel
+        .row
+          .span3
+            %a.btn.btn-primary.btn-primary-muted.btn-claim.connect-twilio-compliant-api{href:"#"}
+              .btn-container
+                .claim-glyph.icon-channel-twilio
+                %p.claim-text
+                  -trans "Connect Twilio Compliant API"
+
+          .span9
+            .claim-description
+              %div
+                -blocktrans
+                  Connect to a Twilio Compliant API such as <a href=https://github.com/dwilkie/adhearsion-twilio>adhearsion-twilio</a>. You can use this to migrate away from Twilio and save on Twilio fees.
+
```

Comment:
Don't link to any provider here, just say Twilio Compliant API. No need for saving fees text etc..